src/rred_reports/reports/report_config.toml

```diff
@@ -0,0 +1,11 @@
+[school]
+template = "input/templates/2021/2021-22_template.docx"
+n_tables = 999
```

Comment:
might be worth having this as the real value rather than a placeholder

Reply:
This is 7 right?